smoketest/scripts/cli/test_interfaces_bonding.py

```diff
@@ -241,6 +241,23 @@ def test_bonding_uniq_member_description(self):
             for member in self._members:
                 self.assertIn(member, slaves)
 
+    def test_bonding_system_mac(self):
+        # configure member interfaces and system-mac
+        system_mac = '00:50:ab:cd:ef:11'
+        for interface in self._interfaces:
+            for option in self._options.get(interface, []):
+                self.cli_set(self._base_path + [interface] + option.split())
+
+            self.cli_set(self._base_path + [interface, 'system-mac', system_mac])
+
+        self.cli_commit()
+
+        # verify config
+        for interface in self._interfaces:
+            slaves = read_file(f'/sys/class/net/{interface}/bonding/slaves').split()
```

Comment:
You verify that the bond members exist as done in many other places here. What‘s missing is the part to verify the system_mac was properly programmed.

Reply:
I've added the right logic.